pkgs/tools/virtualization/guestfs-tools/default.nix

```diff
@@ -0,0 +1,87 @@
+{ lib
+, stdenv
+, fetchurl
+, bison
+, cdrkit
+, cpio
+, flex
+, getopt
+, hivex
+, jansson
+, libguestfs
+, libvirt
+, libxml2
+, makeWrapper
+, ncurses
+, ocamlPackages
+, pcre2
+, perlPackages
+, pkg-config
+, qemu
+, xz
+}:
+
+stdenv.mkDerivation rec {
+  pname = "guestfs-tools";
+  version = "1.48.2";
+
+  nativeBuildInputs = [
+    bison
+    cdrkit
+    cpio
+    flex
+    getopt
+    makeWrapper
+    pkg-config
+    qemu
+  ] ++
+  (with perlPackages; [
+    GetoptLong
+    libintl-perl
+    ModuleBuild
+    perl
+    Po4a
+  ]) ++
+  (with ocamlPackages; [
+    findlib
+    gettext-stub
+    ocaml
+    ocaml_gettext
+    ounit2
+  ]);
+
+  buildInputs = [
+    hivex
+    jansson
+    libguestfs
+    libvirt
+    libxml2
+    ncurses.dev
+    pcre2.dev
+    xz.dev
```

Comment:
Is there a reason to explicitly specify dev outputs here? I remember someone suggested me not to do that unless really necessary in a review I had a while ago.

Reply:
Yeah. This allows the final derivation to be smaller by not linking the unnecessary binaries/manual/etc that are included in those tools.

Reply:
This is unusual. Can we rather remove the references to this in the code?